Create ECSFargateDVWA.json

robinp77 opened the pull request:
This JSON CloudFormation Template provides ECS Cluster, Task Definitions and Service deployment using the DVWA image.

This assumes that you have deployed Vision One Container Security and have the DVWA in your ECR.

## Proposed Changes
  - New blah blah
  - Creates blah blah
  - Full details listed here.

robinp77 committed:
Create ECSFargate&DVWA

This JSON CloudFormation Template provides ECS Cluster, Task Definitions and Service deployment using the DVWA image.

This assumes that you have deployed Vision One Container Security and have the DVWA in your ECR.

AmbientPlatypus commented:
Can you rename the file (to remove the ampersand) and add an extension for readability?

robinp77 committed:
Rename ECSFargate&DVWA to ECSFargateDVWA.json

This JSON CloudFormation Template deploys ECS Cluster, Task Definitions and Service running a single Container instance of DVWA.

This assumes that the Vision One Container Security has already been deployed.
